op-node/rollup/derive/span_batch.go

```diff
@@ -204,9 +207,6 @@ func (bp *spanBatchPayload) decodePayload(r *bytes.Reader) error {
 
 // decode reads the byte encoding of SpanBatch from Reader stream
 func (b *RawSpanBatch) decode(r *bytes.Reader) error {
-	if r.Len() > MaxSpanBatchSize {
-		return ErrTooBigSpanBatchSize
-	}
```

Comment:
So this is now handled by individual `decode...` functions testing against `MaxSpanBatchElementCount` right?

Reply:
This specific limit is checked by the channel_in_reader